lib/audited/auditor.rb

```diff
@@ -285,11 +285,10 @@ def auditing_enabled
 
       def run_conditional_check(condition, matching: true)
         return true if condition.blank?
-
         return condition.call(self) == matching if condition.respond_to?(:call)
-        return send(condition) == matching if respond_to?(condition.to_sym)
+        return send(condition) == matching if respond_to?(condition, include_all: true)
```

Comment:
This actually isn't a kwarg, so just `true` as the second parameter is enough. (this still works since you are creating a new hash here and passing it to the function, and since a hash is non-falsy the check in `respond_to?` still passes, but every check of this function creates an extra object allocation)

Reply:
Sure, good catch. Updated.